packages/babel-cli/test/fixtures/babel/dir --out-dir/stdout.txt

```diff
@@ -1,2 +1 @@
-src/bar/bar.js -> lib/bar/bar.js
-src/foo.js -> lib/foo.js
+🎉  Successfully compiled 2 files with Babel.
```

Comment:
@Andarist, here is the default without `--verbose` flag

Reply:
👍 disregard my comment then